examples/with-edgedb/README.md

```diff
@@ -0,0 +1,134 @@
+$ iwr https://ps1.edgedb.com -useb | iex
+```
+
+### Initialize the EdgeDB project
+
+Initialize the project with the following CLI command:
+
+```bash
+$ edgedb project init
+```
+
+After you follow the prompts, this command will spin up a local EdgeDB instance and apply all the migrations inside `dbschema/migrations`. Now that the project is initialized, all EdgeDB clients initialized inside the project directory will connect to this instance automatically—no need for environment variables or hard-coded configuration. ([Read more about projects here.](https://www.edgedb.com/docs/guides/projects))
+
+### Generate the query builder
+
+This project uses the EdgeQL query builder for TypeScript. This tool can express any EdgeQL query in a code-first way and infers a static return type. Generate it with the following command:
+
+```bash
+$ npx edgeql-js
+```
+
+The query builder consists of several files that are generated into the `dbschema/edgeql-js` directory. Import it like so:
+
+```ts
+import e from './dbschema/edgeql-js'
+```
+
+### Seed the database
+
+```bash
+$ npx ts-node seed.ts
+```
+
+### Start the app
+
+```bash
+$ yarn dev
+```
+
+The application should now be running on http://localhost:3000.
+
+## Notes
+
+### API structure
+
+- `POST /api/post`: Create a new post
+  - Body: `{title: string; content: string; authorName: string}`
+- `PATCH /api/post/:id`: Update a post by `id`
+  - Body: `{title?: string; content?: string;}`
+- `PUT /api/publish/:id`: Publish a post by `id`
+- `DELETE /api/post/:id`: Delete a post by `id`
+
+## Evolving the app
+
+Evolving the application typically requires three steps:
+
+1. Update the schema in `dbschema/default.esdl`
+2. Generate a new migration with `edgedb migration create`
+3. Apply the migration with `edgedb migrate`
+4. Regenerate the query builder with `npx edgeql-js`
+
+## Deployment
+
+To deploy this application, follow the EdgeDB deployment guides for your preferred cloud provider:
+
+- [AWS](https://www.edgedb.com/docs/guides/deployment/aws_aurora_ecs)
+- [Google Cloud](https://www.edgedb.com/docs/guides/deployment/gcp)
+- [Azure](https://www.edgedb.com/docs/guides/deployment/azure_flexibleserver)
+- [DigitalOcean](https://www.edgedb.com/docs/guides/deployment/digitalocean)
+- [Fly.io](https://www.edgedb.com/docs/guides/deployment/fly_io)
+- [Docker](https://www.edgedb.com/docs/guides/deployment/docker) (cloud-agnostic)
+
+After setting up your cloud-hosted EdgeDB instance:
+
+1. Find your instance's DSN (AKA connection string). The exact instructions for this depend on which cloud you are deploying to.
+2. Deploy this app to Vercel with the button below. You'll be prompted to provide a value for `EDGEDB_DSN`.
+3. Open the application.
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/with-edgedb&project-name=with-edgedb&repository-name=with-edgedb&env=EDGEDB_DSN)
```

Comment:
```suggestion
```
I don't think we want to have the deploy button twice in the README as we only have it once in our other examples.